packages/bpk-component-accordion/src/BpkAccordionItem.module.scss

```diff
@@ -57,7 +57,7 @@
   }
 
   &__title-text {
-    display: inline-block;
+    display: inherit;
```

Comment:
`display: inline-block` doesn't make sense anyway as it is within the class above which has `display: inline-flex`

Reply:
What `display` value is `inherit`ed?

Reply:
`display: inline-flex;` [from this line](https://github.com/Skyscanner/backpack/pull/3297/files#diff-5b1965da520f4d42beb4451d6dabd6724b1b130710c78ca5ced991e1709ce67aR52)

Reply:
From Mungo's advice, changed to `display:block` with change to tagName to h3